XPlat/Google.Grpc/build.cake

```diff
@@ -80,7 +41,7 @@ Dictionary<string, string> JAR_URLS_ARTIFACT_FILES= new Dictionary<string, strin
 	},
 };
 string ARTIFACT_FILE = "";
-string NUGET_VERSION=$"{ARTIFACT_VERSION}.2";
+string NUGET_VERSION=$"{ARTIFACT_VERSION}";
 
 
 BuildSpec buildSpec = new BuildSpec ()
```

Comment:
We should probably move away from this BuildSpec and to standard cake: https://github.com/xamarin/XamarinComponents/pull/684/files#diff-c578a2377ba3bb7693c87d13619d81c4

Reply:
I moved away from BuildSpec to standard cake on all of my PRIVATE projects. initially i wanted to have the same Cake "framework" for Xamarin/microsoft and my stuff, but the fight with keeping Cake work with .NET core, mono versions and various libraries it was too much wasted time. So I switched to standard cake.

I had few bindings here and in other repos with vanilla Cake, but they were rewritten to BuildSpec, so I understood that `BuildSpec` is way to go.

Reply:
I'm just trying to bump the bindings, so I can continue working on AndroidX updates.

Let's talk about vanilla Cake in the meeting and decide what is the best and after decision I will rewrite scripts.

IMO moving away from BuildSpecs is good idea. But I d like to finish some other priorities 1st.